target/linux/pistachio/base-files/lib/functions/mac.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
all the mac address related scripts and functions should be moved into different commit as its not part of Bluetooth scripts